packages/frontend/src/creation-form/components/outcomes-configuration/single-configuration/index.tsx

```diff
@@ -67,7 +69,11 @@ export const SingleConfiguration = ({
             )}
             <div className="flex justify-between items-center">
                 <Typography>{t("label.binary")}</Typography>
-                <Switch checked={binary} onChange={handleBinaryChange} />
+                <Switch
```

Comment:
We could maybe remove the switch altogether to avoid confusion?

Reply:
Fixed in 450e0c385e0ffdafeb988234e34c53cf7ded5822.